.github/workflows/ci.yaml

```diff
@@ -347,6 +347,11 @@ jobs:
         with:
           path: compiler/parser/python.rs
           key: lalrpop-${{ hashFiles('compiler/parser/python.lalrpop') }}
+      - name: spell checker
```

Comment:
I'm perfectly fine with this since I have cspell locally but it will impose an additional step on new contributors.

Reply:
how we can prevent spell regression while keeping it less burden? do you have a good idea?

Reply:
Ideally, a github bot could exist for this. Other than that, not sure, we can either do it manually (as you did in the previous PR) or just merge this and see if it is commonly a point of frustration for new contributors.

Reply:
Alternatively, we could use a pull request template to make sure that contributors have done the necessary routine items we expect. For example:

```markdown
<!-- Use [x] to mark item done, or just click the checkboxes with device pointer -->

- [ ] Did you check to see if everything is spelled correctly, using [cspell](https://github.com/streetsidesoftware/cspell) and/or its [VSCode extension](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker)?
  - If you are introducing any new spellings, please let the maintainers know.
- [ ] Did you format your code using `cargo fmt`?
- [ ] Did you lint your code using `cargo clippy`?

-----
```

Reply:
Instead of a list that requires manual ticking, just link to `DEVELOPMENT.md#code-style` that can mention these?